torch/csrc/api/include/torch/nn/functional/loss.h

```diff
@@ -6,15 +6,33 @@ namespace torch {
 namespace nn {
 namespace functional {
 
+inline Tensor l1_loss(
+    const Tensor& self,
+    const Tensor& target,
+    const L1LossOptions& options) {
```

Comment:
I think we should do
```suggestion
    const L1LossOptions& options = {}) {
```
so that users can call this without passing options. And we should add a test for the `l1_loss(input, target)` use case too. (And same for all other functionals in this PR.)